### PR TITLE
feat: add examples of iv curves

### LINKED ISSUE #262: Example with IV and annotations
@maasencioh Please could you add a full example in Experimental Spectra section of the storybook of your data ?

The goal for now is to put the data and the annotations to be sure nothing is missing (just display, no interactions).

In the future we would still add the zoom and interactivity but this is something @wadjih-bencheikh18 is currently working on.

### PR DESCRIPTION
**Pull request by maasencioh**
closes #262

### CODE REVIEW

**Comment by targos** [comment]
Can you fix the type issue?

**Comment by lpatiny** [comment]
@maasencioh could you check that the CI is passing the tests ?

**Comment by maasencioh** [comment]
Is now passing the test, but as soon as https://github.com/cheminfo/analysis/issues/8 is fixed I'll migrate to `iv-analysis`

**Review by targos — DISMISSED** [review]
waiting for migration to the new package

**Comment by maasencioh** [comment]
Thank you @targos 👏

File: tsconfig.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "removeComments": false,
     "sourceMap": false,
     "strict": false,
-    "target": "es2019"
+    "target": "es2019",
+    "skipLibCheck": true
```
Comment (targos) [inline tsconfig.json:15]: Can you cancel this now?

**Pull request merged**